frontend/app/templates/create-badges.hbs

```diff
@@ -1,3 +1,33 @@
+<script>
```

Comment:
Please move the script to the end of the file.

Reply:
Will Do 👍